### PR TITLE
Folders Restructured, Login page Completed with Responsive Design

### PR DESCRIPTION
**Pull request by DilshanHJ**

### CODE REVIEW

File: src/Pages/Auth/OTPVerify.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import otp_clipArt from "../../assets/Images/OTP_clipArt.png";
+import { useState } from "react";
+const OTPVerify = () => {
+  const [OTP, setOTP] = useState("");
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [phone, setPhone] = useState("071-1234567");
+  return (
+    <div className=" flex flex-row w-full h-screen bg-second">
+      <div className=" max-md:w-full h-full min-w-[340px] w-1/2 bg-second flex flex-col justify-center items-center">
+        <form
```
Comment (sachin96Boy) [inline src/Pages/Auth/OTPVerify.tsx]: let's keep form sections as separate components.  relevant logics on top sections can be called within those sub-components.

File: src/Pages/Auth/OTPVerify.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <ResendOTP
+              className="font-[700] font-montserrat text-[13px] text-tertiary-alt"
+              onResendClick={() => console.log("Resend clicked")}
+            />
+          </p>
+
+          <button
+            type="submit"
+            onClick={(e) => {
+              e.preventDefault();
```
Comment (sachin96Boy) [inline src/Pages/Auth/OTPVerify.tsx]: Let's handle forms with Formik or react hooks form, for validation try Yup or Zod libraries

File: src/Pages/Auth/ProfileUpdate.tsx [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+            backgroundImage: `url(${register_image})`,
+            backgroundPosition: "center",
+            backgroundSize: "cover",
+          }}
+          className="h-full w-full rounded-[10px]"
+          alt=""
+        />
+      </div>
+      <div className=" max-md:w-full min-h-full min-w-[340px] w-full bg-second flex flex-col justify-center items-center p-4">
+        <form
```
Comment (sachin96Boy) [inline src/Pages/Auth/ProfileUpdate.tsx]: Let's try form handling as mentioned above

File: src/Pages/Auth/ProfileUpdate.tsx [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+              {}
+            </p>
+          </div>
+          <div>
+            <p className="font-[500] font-montserrat text-[15px] text-prime-alt w-fit text-center ">
+              Upload images
+            </p>
+            <div className="flex flex-row justify-start items-start gap-x-8 gap-y-4 w-full flex-wrap mt-2">
+              <div className="flex flex-col justify-center items-center gap-x-4 w-fit">
+                <Dropzone
```
Comment (sachin96Boy) [inline src/Pages/Auth/ProfileUpdate.tsx]: try moving the dropzone logic to a custom component and handle that logic seperaly

File: src/Pages/Auth/login.tsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+import login_image from "../../assets/Images/login_banner.png";
+import TextInput from "../../Components/Elements/TextInput";
+import { useState } from "react";
+export default function Login() {
+  const [phone, setPhone] = useState("");
+  const [password, setPassword] = useState("");
+  return (
+    <div className=" flex flex-row w-full h-screen bg-second">
+      <div className=" max-md:w-full h-full min-w-[340px] w-1/2 bg-second flex flex-col justify-center items-center">
+        <form
```
Comment (sachin96Boy) [inline src/Pages/Auth/login.tsx]: Try form handling as mentioned above in comments

File: src/Pages/Auth/register.tsx [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+            backgroundImage: `url(${register_image})`,
+            backgroundPosition: "center",
+            backgroundSize: "cover",
+          }}
+          className="h-full w-full rounded-[10px]"
+          alt=""
+        />
+      </div>
+      <div className=" max-md:w-full min-h-full min-w-[340px] w-full bg-second flex flex-col justify-center items-center p-4">
+        <form
```
Comment (sachin96Boy) [inline src/Pages/Auth/register.tsx]: form handling as mentioned above

File: src/App.tsx [hunk 7]
```diff
@@ -1,25 +1,41 @@
...
+          <Route path="profile" element={<Profile />} />
+          <Route path="payment" element={<h1>Payment</h1>} />
+          <Route path="contact" element={<h1>Contact</h1>} />
+          <Route path="shopping" element={<Shopping />} />
+          <Route path="notifications" element={<h1>Notifications</h1>} />
+        </Route>
+        <Route path="login" element={<Login />} />
+        <Route path="register" element={<Register />} />
+        <Route path="otp-verify" element={<OTPVerify />} />
+        <Route path="profile-update" element={<ProfileUpdate />} />
```
Comment (sachin96Boy) [inline src/App.tsx:38]: add a reroute Route path that returns to home if path is different

**Review by sachin96Boy — COMMENTED** [review]
Review added

File: src/Pages/Auth/OTPVerify.tsx (hunk 1 above)
Comment (DilshanHJ) [reply src/Pages/Auth/OTPVerify.tsx]: done

File: src/Pages/Auth/OTPVerify.tsx (hunk 2 above)
Comment (DilshanHJ) [reply src/Pages/Auth/OTPVerify.tsx]: done

File: src/Pages/Auth/ProfileUpdate.tsx (hunk 3 above)
Comment (DilshanHJ) [reply src/Pages/Auth/ProfileUpdate.tsx]: done

File: src/Pages/Auth/ProfileUpdate.tsx (hunk 4 above)
Comment (DilshanHJ) [reply src/Pages/Auth/ProfileUpdate.tsx]: done

File: src/Pages/Auth/login.tsx (hunk 5 above)
Comment (DilshanHJ) [reply src/Pages/Auth/login.tsx]: done

File: src/Pages/Auth/register.tsx (hunk 6 above)
Comment (DilshanHJ) [reply src/Pages/Auth/register.tsx]: done

File: src/App.tsx (hunk 7 above)
Comment (DilshanHJ) [reply src/App.tsx:38]: done

**Pull request merged**